UnityProject/Assets/Scripts/Chat/ChatRelay.cs

```diff
@@ -205,4 +190,27 @@ private void UpdateClientChat(string message, ChatChannel channels)
 			ChatUI.Instance.AddChatEntry(message);
 		}
 	}
+
+	/// <summary>
+	/// Sends a message to TTS to vocalize.
+	/// Only messages that are spoken by player characters are currently vocalized.
+	/// Messages must contain at least one letter from the alphabet.
+	/// </summary>
+	/// <param name="message">The message to try to vocalize.</param>
+	private void trySendingTTS(string message)
```

Comment:
Methods should be PascalCase. Remember for next time

Reply:
Whoops! Will do.

Feel free to request changes next time I forget 😙